setup.cfg

```diff
@@ -3,3 +3,7 @@ testpaths = pymc3/tests
 
 [coverage:run]
 omit =  *examples*
+
+[pydocstyle]
+add-ignore = D100,D104
```

Comment:
I'll add a comment -- D100 wants a docstring at the top of every file, and D104 wants a docstring in every `__init__.py` file